modules/op-installing-crunchy-postgres-database-and-tekton-hub.adoc

```diff
@@ -0,0 +1,168 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="installing-crunchy-postgres-database-and-tekton-hub_{context}"]
+= Optional: Installing Crunchy Postgres database and {tekton-hub}
+
+[role="_abstract"]
+Cluster administrators can install the Crunchy Postgres database and configure {tekton-hub} to use it instead of the default database. 
+
+[discrete]
+.Prerequisites
+
+* Install the Crunchy Postgres Operator from the Operator Hub.
+* Create a Postgres instance that initiates a Crunchy Postgres database.
+
+[discrete]
+.Procedure
+
+. Get into the Crunchy Postgres pod.
+
+.Example: Getting into the `test-instance1-m7hh-0` pod 
++
+[source,terminal]
+----
+$ oc exec -it -n openshift-operators test-instance1-m7hh-0 -- /bin/sh
+
+Defaulting container name to database.
+Use 'oc describe pod/test-instance1-m7hh-0 -n openshift-operators' to see all of the containers in this pod.
+sh-4.4$ psql -U postgres
+psql (14.4)
+Type "help" for help.
+----
+
+. Find the `pg_hba.conf` file.
```

Comment:
@sounix000 The step order is not correct here. Please check the preview and correct it. I can see the + sign in the preview for L24. Needs to be removed.